yarn-project/noir-contracts/src/contracts/pokeable_token_contract/src/main.nr

```diff
@@ -79,16 +56,20 @@ contract PokeableToken {
         // Should eventually be hidden:
         inputs: PrivateContextInputs,
         //*********************************/
+        sender: Field,
+        recipient: Field
     ) -> distinct pub abi::PrivateCircuitPublicInputs {
-        let mut context = Context::new(inputs, 0); // Since there are no arguments, we can just 0 as the args hash.
+        let mut context = Context::new(inputs, abi::hash_args([sender, recipient]));
         let storage = Storage::init();
 
-        let sender = storage.sender.get_note(&mut context);
-
-        let recipient = storage.recipient.get_note(&mut context);
-
+        // TODO: This check is not satisfying constraints
```

Comment:
Out of curiosity do you know why this doesn't work?

Reply:
Not really, I timed out trying to troubleshoot it